tests/cypress/plugins/index.js

```diff
@@ -31,6 +32,27 @@ module.exports = (on, config) => {
         },
     });
     on('task', { isFileExist });
+    on('task', {
+        createFolder: (folder) => {
+            if (!fs.existsSync(folder)) {
+                fs.mkdirSync(folder);
+            }
+            return null;
```

Comment:
Just curios about why we return ``null`` every time.

Reply:
A task must not return undefined, so returning null is quite a common practice as I see in the [docs](https://docs.cypress.io/api/commands/task#Syntax)